apps/geo/dataloaders.py

```diff
@@ -32,6 +32,15 @@ def batch_load_fn(self, keys):
         return Promise.resolve([_map.get(key) for key in keys])
 
 
+class GeoAreaLoader(DataLoaderWithContext):
+    def batch_load_fn(self, keys):
+        geo_area_qs = GeoArea.objects.filter(id__in=keys).defer('polygons', 'centroid', 'cached_data')
```

Comment:
Let's use `get_geo_area_queryset_for_project_geo_area_type` to include annotated fields used by **ProjectGeoAreaType**. Let's add a note as well about this here.
Also, let's move this to assessment_registry dataloader instead of geo/

Also, let's update get_geo_area_queryset_for_project_geo_area_type default defer_fields to include cached_data.